### DISCUSSION TITLE
401 Error using correct token

### DISCUSSION THREAD

**Discussion by nakedeye515** [dgreif/ring#706] [Q&A]
I previously had everything working fine.  Camera would preview and show live when clicked on in the home app.  Then it would only show the prevue in the app and time out when you would click the camera.  In my attempt to fix it, I purged out  the plugin and readded it, ran the cli commands to get my token and added it. I am getting a constant 401 error whenever I try to load the plugin.  I know I have the right token because I get a very different error than the one I have pasted below.  Suggestions?

[Ring] HTTPError: Response code 401 (Unauthorized)
    at Request.<anonymous> (/home/hoobs/.hoobs/node_modules/got/dist/source/as-promise/index.js:117:42)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: undefined,
  timings: {
    start: 1627427682624,
    socket: 1627427682796,
    lookup: 1627427682797,
    connect: 1627427682797,
    secureConnect: 1627427682797,
    upload: 1627427682802,
    response: 1627427682919,
    end: 1627427682934,
    error: undefined,
    abort: undefined,
    phases: {
      wait: 172,
      dns: 1,
      tcp: 0,
      tls: 0,
      request: 5,
      firstByte: 117,
      download: 15,
      total: 310
    }
  }
}
7/27/2021, 7:14:42 PM [Ring] Request to https://app.ring.com/rhq/v1/devices/v1/locations failed:
7/27/2021, 7:14:42 PM [Ring] Error connecting to API

**Comment by dgreif** [comment]
Looks like Ring removed the endpoint we were using before. Fixed in 9.18.6
